tests/misc/test_file_server.py

```diff
@@ -0,0 +1,13 @@
+import subprocess
+
```

Comment:
Even if the requirements are simple, please add a "requirements" comment on top.